<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupGpi4</class>
 <widget class="QWidget" name="SetupGpi4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Управління</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>559</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2" colspan="2">
    <widget class="QPushButton" name="Close">
     <property name="text">
      <string>Закрити</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="bnOk">
     <property name="text">
      <string>Виконати</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Управління</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Автоматичний пуск</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="zdMode">
        <item>
         <property name="text">
          <string>Виключено</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Один цикл</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>По штанзі</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Періодична загрузка</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Настройки алгоритмів</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Затримка на провисання штанги</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="zdTProbeWait">
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="maximum">
         <double>60.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Вагонеток на годину</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>хв</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="nomLoad">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>Завдання</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="zdGst">
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>кг</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_42">
        <property name="text">
         <string>кг</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_15">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Вага вугілля</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="zdGcl">
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_38">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Вага шихти</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="pGclst">
        <property name="keyboardTracking">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_39">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Співвідношення  вугілля/шихта</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_43">
        <property name="text">
         <string>%</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>zdGcl</tabstop>
  <tabstop>pGclst</tabstop>
  <tabstop>zdGst</tabstop>
  <tabstop>zdTProbeWait</tabstop>
  <tabstop>zdMode</tabstop>
  <tabstop>bnOk</tabstop>
  <tabstop>Close</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
